reckless_lib/src/utils.rs

```diff
@@ -58,6 +58,9 @@ pub fn clone_recursive_fix(repo: git2::Repository, url: &str) -> Result<(), Reck
     Ok(())
 }
 
+pub fn get_repo_name_from_url(url: &str) -> &str {
+    url.split("/").last().unwrap()
```

Comment:
```suggestion
    url.split("/").last().unwrap().strip_suffix(".git")
```

You should remove the `.git`, you may want to use the https://doc.rust-lang.org/std/string/struct.String.html#method.strip_suffix

Reply:
Done.